format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
We are currently reserving `self`, `related` and 4 pagination links. I support the complete openness of the spec to the future in this regard, please review my updated PR.